math_science_reqs/36226.md

```diff
@@ -0,0 +1,32 @@
+# 36-226: Statistical Inference
+|  Category|Difficulty  |
+|--|--|
+|Homework  | 6 |
+|Exams  | 5 |
+
+This course is the second semester of a year-long course in probability and statistics. It builds on the ideas of random variables, probability distributions, expectation, etc. from the prerequisite classes (36-225 or 36-219) with an emphasis on methods of analyzing data and making inferences about an unknown population based on information contained in a sample. Topics include methods and properties of estimation, hypothesis testing, linear models, and analysis of variance. 
+
+
+## What to Expect
+-   Homework: Homeworks are mostly just written problem sets with the occasional programming problem, which is done in R. You don’t need any familiarity with R because they generally teach everything you need to know. The written problems are pretty heavy on the theory and often include proofs/derivations.
+    
+-   Exams: There is a general consensus that the exams are easier than the homework. The difficulty of exam problems is comparable to that of recitations and exam review problems.
+
+## How to do well
+-   Go to class! Many of the concepts will be new since this course provides a rigorous mathematical background to statistical concepts.
+    
+-   Read the class notes before attempting the homework. The problems on the HW are challenging and require a thorough understanding of the material to complete them.
+    
+-   Review material from the pre-requisite classes. The class assumes familiarity with material from 36-219/225 and thus skims over the review pretty quickly. You should be very comfortable with random variables, discrete/continuous distributions, expectation, conditional probability, etc.
+
+## What to watch out for
+-   The homeworks start on the easier side and get harder as the material gets more complex. Make sure that you are on top of the material and get your questions answered as soon as possible.
+    
+-   The recitations are quite hands-off as the TAs allow you to work on the problems on your own for the majority of the time. Be sure to utilize office hours if you don’t find recitation helpful. But whether you go to recitation or not, it is imperative that you do the recitation problems before the exam!
+
+## Resources
+-   Office hours
+    
+-   Recitations
+    
+-   Exam reviews

```

Comment:
Can you elaborate on each one, what they are about, and why they are useful to go to.

Are there any textbooks that are used in the class or extra materials that were helpful? Course website?